src/particles/transformation/TO_FIXED_T.H

```diff
@@ -23,20 +23,23 @@ namespace impactx
 {
 namespace transformation
 {
-    struct Z2T
+    struct To_Fixed_T
     {
         using PType = ImpactXParticleContainer::ParticleType;
 
-        /** A Z-to-T transformation
+        /** Transformation of particles from fixed location s to fixed time
+         *  t.  At fixed t, each particle is represented by phase space
```

Comment:
Doxygen: title line + empty newline + body
```suggestion
        /** Transformation of particles from fixed location s to fixed time t.
         *
         * At fixed t, each particle is represented by phase space
```